modules/ROOT/pages/deploy-on-premises.adoc

```diff
@@ -213,6 +215,33 @@ include::example$mmp-concept-config.xml[]
 mvn clean package deploy -DmuleDeploy
 ----
 
+== Deploy Applications in Parallel
+
+You can deploy applications in parallel to on-premises Mule instances. Deploying applications in parallel reduces the startup time when you deploy a large number of apps. To prevent deployment failure, ensure that applications are not dependent on each other because a particular start order cannot be guaranteed.
+
+To enable parallel deployment:
+
+. Package or export your Mule applications to the `/apps` directory of your Mule runtime engine instance.
+. Start Mule using the `-M-Dmule.deployment.parallel` option:
++
+```
+mule -M-Dmule.deployment.parallel
+```
+
+After performing these steps, all applications in the `/apps` directory start in parallel.
+
+== Update Applications at Runtime
+
+Updating a Mule application at runtime can be a complex change involving class modifications, endpoint modifications (for example, changing ports, etc), as well as reconfiguring flows. As a result, any application update does a graceful app shutdown under the hood and reconfigures itself. In practice, this is pretty transparent to the user and happens within seconds.
+
+There are several ways you can update an application:
+
+* By dropping the modifications over an existing exploded app folder and touching the master configuration file (`mule-config.xml` located in the app's root directory by default).
+* By dropping a new `jar` version of the app into the `$MULE_HOME/apps` directory. Mule detects the `jar` as an existing application and ensures the updated by a clean redeployment of the app. +
```

Comment:
Per our recommended writing style, break list items and steps after one sentence:
```suggestion
* By dropping a new `jar` version of the app into the `$MULE_HOME/apps` directory. 
Mule detects the `jar` as an existing application and ensures the updated by a clean redeployment of the app. +
```
Also, a couple of questions about the meaning:
"a new `jar` version of the app: is this a JAR file replacement for the app? The app is in a JAR file?
Does Mule detect this as an UPDATE and overwrite the existing app file?